config/schema/field_types.json

```diff
@@ -1,6 +1,6 @@
 {
   "identifier": {
-    "description": "An identifier, can be used for exact (case and whitespace sensitive) lookup and facetting",
+    "description": "An identifier, can be used for exact (case and whitespace sensitive) lookup and aggregating",
```

Comment:
I would agree with the keeping faceting as the internal and external feature name. Aggregations seem to be an elasticsearch abstraction, and feel more like a implementation detail to rummager.

As Mat says faceting has an established meaning. Keeping the rummager faceting feature naming nicely separates the use case (faceted search) from the implementation (which uses the `elasticsearch facets` feature now, will start using the `aggregations` feature with this PR, but could use another implementation in the future).